content/docs/buildpacks/language-family-buildpacks/ruby.md

```diff
@@ -37,29 +37,34 @@ notes](https://github.com/paketo-buildpacks/ruby/releases/latest).
 ## Specifying a Ruby Version
 
 The Ruby Buildpack allows you to specify a version of Ruby to use during
-deployment. This version can be specified via `buildpack.yml` or a `Gemfile`.
-When specifying a version of Ruby, you must choose a version that is available
+deployment. This version can be specified via the `BP_MRI_VERSION` environment
+variable or a `Gemfile`. When specifying a version of Ruby, you must choose a version that is available
 within the buildpack. The supported versions can be found
 [here](https://github.com/paketo-buildpacks/mri/releases/latest).
 
+Please note that setting the Ruby version through a `buildpack.yml` file will
+be deprecated in MRI Buildpack v1.0.0.
+
+will be deprecated in Bundler v1.0.0.
```

Comment:
This seems like some left over copy. You will probably want to remove this or edit it to make what you are trying to get across clearer.